models/bpf.go

```diff
@@ -0,0 +1,132 @@
+// Copyright Contributors to the L3AF Project.
+// SPDX-License-Identifier: Apache-2.0
+
+package models
+
+import (
+	"github.com/cilium/ebpf"
+	"github.com/l3af-project/l3afd/config"
+)
+
+// BPF defines run time details for BPFProgram.
+type BPF interface {
+	// Name returns the BPFProgram Name
```

Comment:
Based on https://github.com/l3af-project/l3afd/pull/199, I think this is the package name not the program name.